src/java/com/unifina/signalpath/blockchain/EthereumModuleOptions.java

```diff
@@ -64,6 +64,14 @@ private void readNetworkOption(ModuleOptions options) {
 		}
 	}
 
+	public String getDatacoinAddress(){
```

Comment:
Not related to Ethereum module options, probably wrong place

Reply:
Yeah, but where should I put this method? A new class like com/unifina/signalpath/blockchain/StreamrOptions.java ?